src/prop/minisat/core/Solver.cc

```diff
@@ -607,6 +607,20 @@ Lit Solver::pickBranchLit()
             << "getNextTheoryDecisionRequest(): now deciding on " << nextLit
             << std::endl;
         decisions++;
+
+        // org-mode tracing, including propagations is a separate trace
+        Trace("dtview")
+            << std::string(context->getLevel()
+                               - (options::incrementalSolving() ? 1 : 0),
+                           '*')
+            << " " << proxy->getNode(MinisatSatSolver::toSatLiteral(nextLit))
+            << " :THEORY-DECISION:" << std::endl;
+        Trace("dtview::prop")
+            << std::string(context->getLevel() + 1
+                               - (options::incrementalSolving() ? 1 : 0),
+                           '*')
+            << " /Propagations [Last Decision Repeated]/" << std::endl;
+
```

Comment:
How about wrapping these trace statements in a function to reduce the code duplication?

Reply:
Thanks for the feedback! Does something like this look okay? https://github.com/CVC4/CVC4/blob/4f4cc1ac23b9cef207fd21d924b4e74508b75e16/src/prop/minisat/core/Solver.h#L460